manual-tests.md

```diff
@@ -231,3 +231,19 @@
 
     * Check that it should end ok
     * Check in the Web that the description is now what the YAML says
+
+
+# Test push binary metadata with conflicts
+
+1. 'snapcraft build' a simple snap
+2. Do a simple 'snapcraft push SNAP'
+3. Go to the Web and change snap's icon
+4. Change the snap's icon in the YAML file to something different than you put in the Web
+5. Try to update snap's metadata using `snapcraft push-metadata SNAP`
```

Comment:
Does pushing the snap also raise the conflict?

Reply:
Can we add a test for that?

Reply:
Push won't raise the conflict, and it won't update the metadata either. Metadata from a pushed revision will only be used if there are no previous changes made in the web UI.
Should we think about adding some warning/message while processing a push about conflicting metadata?

Reply:
We shouldn't block on this for the PR, but an error code saying the metadata is not being updated due to divergences for which we can on the client instruct the user would be good. Being silent by default is fine.